[2024-12-11T11:14:27.659+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-11T11:14:27.675+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_pl.etl_dim_user manual__2024-12-11T11:12:18.253285+00:00 [queued]>
[2024-12-11T11:14:27.685+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_pl.etl_dim_user manual__2024-12-11T11:12:18.253285+00:00 [queued]>
[2024-12-11T11:14:27.685+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-11T11:14:27.700+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_user> on 2024-12-11 11:12:18.253285+00:00
[2024-12-11T11:14:27.708+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2337) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-11T11:14:27.710+0000] {standard_task_runner.py:72} INFO - Started process 2339 to run task
[2024-12-11T11:14:27.709+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_pl', 'etl_dim_user', 'manual__2024-12-11T11:12:18.253285+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/test_etl_dag.py', '--cfg-path', '/tmp/tmp9nbmqtm9']
[2024-12-11T11:14:27.711+0000] {standard_task_runner.py:105} INFO - Job 134: Subtask etl_dim_user
[2024-12-11T11:14:27.763+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_pl.etl_dim_user manual__2024-12-11T11:12:18.253285+00:00 [running]> on host 751be410a86b
[2024-12-11T11:14:27.863+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_pl' AIRFLOW_CTX_TASK_ID='etl_dim_user' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T11:12:18.253285+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-11T11:12:18.253285+00:00'
[2024-12-11T11:14:27.865+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-11T11:14:27.897+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-11T11:17:14.365+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-11T11:17:14.376+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-11T11:17:14.377+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_pl, task_id=etl_dim_user, run_id=manual__2024-12-11T11:12:18.253285+00:00, execution_date=20241211T111218, start_date=20241211T111427, end_date=20241211T111714
[2024-12-11T11:17:14.426+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-11T11:17:14.442+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-11T11:17:14.448+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
